leakcanary-analyzer/src/main/java/com/squareup/leakcanary/ShortestPathFinder.java

```diff
@@ -176,6 +176,13 @@ private void visitRootObj(LeakNode node) {
     RootObj rootObj = (RootObj) node.instance;
     Instance child = rootObj.getReferredInstance();
 
+    //We should ignore leaks where an android.os.Binder is the root of the leak.
+    //When you bind and unbind from a Service, the OS will tend to keep a reference to the Binder
+    // after we'd expect it to be collected.
+    if (HahaHelper.extendsFrom(child, "android.os.Binder")) {
```

Comment:
mmm what I just said won't work, you only want to ignore roots. Well, we could have a new method that's `ExcludedRefs.Builder.rootSubclass()` and ignores leaks where the root is a subclass of the provided class.